ddp-workspace/projects/ddp-pancan/src/app/components/page-with-sections/page-with-sections.component.html

```diff
@@ -1,5 +1,5 @@
 <main class="page" [ngClass]="isAboutUsPage ? 'about-us': 'scientific-research'">
-    <section *ngFor="let section of (source + '.Sections') | translate; let i = index;" [class]="'section section-' + i">
+    <section *ngFor="let section of (source + '.Sections') | translate; let i = index;" [class]="'section section-' + i "[id] = "'section-' + i">
         <div class="section__content">
             <h3 class="section__title static-page-title">{{section.Title}}</h3>
```

Comment:
We can keep the `i` var where it is above, but add the `id` attribute to `h3` so that the very top of the page is the title instead of the whitespace above it.
```suggestion
            <h3 class="section__title static-page-title" [id] = "'section-' + i">{{section.Title}}</h3>
```